README.md

```diff
@@ -1,3 +1,55 @@
-# SwornDisk v2
+# MlsDisk
 
-SwornDisk v2 adopts on a new layered design and implementation.
+## Introduction
+
+MlsDisk is a **multilayered log-structured secure virtual disk for TEEs**, which has the following key features:
+
+* **Transparent protection.** As a virtual disk, MlsDisk can _transparently_ protect any file system (e.g., Ext4) that is stacked upon it and runs inside a TEE from a strong adversary outside the Trusted Execution Environment (TEE).
+
+* **Strong security.** MlsDisk promises six security properties: _confidentiality_, _integrity_, _freshness_, _consistency_, _atomicity_, and _irreversibility_. For more information, see [the security goal](#security-goal) below.
+
+* **High performance.** Thanks to its unique log-structured, amplification-minimized design, MlsDisk can deliver an excellent I/O performance that is close to the theoretically optimal level.
```

Comment:
Without mentioning read/write amplification first, the reader may be confused as to what "amplification-minimized" means. And I haven't seen this term before. Is it a widely accepted term?

I suggest changing the sentence to the following form. 

> Thanks to its unique multilayered log-structured design, MlsDisk can ...

Or if you feel it important to give the reader more hints as to why MlsDisk can performance well. Here is another way to rewrite the sentence.

> MlsDisk introduces a unique multilayered log-structured approach
> that achieves the security goal with minimal I/O amplification,
> delivering an excellent I/O performance that is close to the theoretically optimal level.